beta-src/src/components/ui/WDPillScroller.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import { Box, ButtonGroup } from "@mui/material";
+import WDScrollButton from "./WDScrollButton";
+import { gameStateProps } from "../../interfaces/PhaseScroll";
+import { ScrollButtonState } from "../../enums/UIState";
+
+const WDPillScroller: React.FC<gameStateProps> = function ({
+  disabled = false,
+  gameState,
+  onChangeSeason,
+}): React.ReactElement {
+  return (
+    <Box
+      sx={{ alignItems: "center", display: "flex" }}
+      style={{ filter: "drop-shadow(0px 8px 9px black)" }}
+    >
+      <ButtonGroup>
+        <WDScrollButton
+          direction={ScrollButtonState.BACK}
```

Comment:
Opposite of `forward` is `backward`, opposite of `front` is `back`

Reply:
LOL clearly directions aren't my strong suit.